src/Chapter2.hs

```diff
@@ -620,7 +630,11 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven [x, _, xs] = [x, xs]
```

Comment:
This line is actually redundant. It is equivalent to the following pattern:

```haskell
takeEven (x : _ : xs : [])
```

Yes, in your case you use square brackets, and variable `xs` is actually a list element. But, again, this case is covered in two steps by the next and previous patterns. No need to have extra pattern :slightly_smiling_face: